### ISSUE TITLE
TACC Symposium call for abstracts: TACC Symposium for Texas Researchers September 23-24, 2021 Inbox

### ISSUE THREAD

**Issue by ewommack** [ArctosDB/arctos#3809]
TACC Symposium for Texas Researchers September 23-24, 2021
From: Joshua Freeze
We are pleased to announce the fourth TACC Symposium for Texas Researchers (TACCSTER) being held on September 23-24, 2021.

TACCSTER is a meeting of scientists, engineers and scholars from across the state of Texas who use, and would like to use, the Texas Advanced Computing Center (TACC) to advance their research goals. This meeting will be an opportunity for you to showcase your ongoing research involving TACC resources, attend and participate in Birds-of-a-Feather panel discussions, learn new and useful computing skills from a variety of tutorials, network with other Texas-based researchers, and hear information about the latest TACC systems and services.

The meeting will be held on September 23-24, 2021 virtually and at the Texas Advanced Computing Center in Austin, Texas.

To register and for more information please visit: https://www.tacc.utexas.edu/taccster

Please contact taccster@tacc.utexas.edu if you have any questions.

To manage your TACC Announcements subscriptions, please visit https://www.tacc.utexas.edu/news/subscribe

Deadline for poster abstracts is 20th of August. Have we ever attended before?

**Comment by Jegelewicz** [comment]
I believe that @mvzhuang went once? Maybe we can get someone from Angelo State to go this year?

**Comment by mvzhuang** [comment]
Yes-- presented a 5 min version of the “Arctos: A Collaborative Collection Management Solution” talk adapted from the “Arctos: A Tool to Help Small Collections Make Their Case” and the Arctos poster.

**Comment by ewommack** [comment]
How was it @mvzhuang? Did it seem worthwhile to go?

**Comment by campmlc** [comment]
Also might be good to get a Berkeley person there, to make contacts and
ensure we have the resources we need for SABI for mirrors/backups etc?

**Comment by Jegelewicz** [comment]
> get a Berkeley person there

This conference is for TEXAS researchers!

**Comment by campmlc** [comment]
Oh, I thought Tom G attended a couple years ago. That must have been
something else. So not people using TACC data for research?

**Comment by Jegelewicz** [comment]
> TACCSTER is a meeting of scientists, engineers and scholars **from across the state of Texas** who use, and would like to use, the Texas Advanced Computing Center (TACC) to advance their research goals.

**Comment by mvzhuang** [comment]
It was interesting, but 95% of it flew over my head and it was pretty hard core computer science projects and oohing over supercomputer specs. There are one-on-one’s available and I think workshops on allocations etc.. Pretty sure, they wouldn’t mind a non-Texas rep from Arctos since there’s a virtual component and if they represented Texas researchers? Maybe? I think I was a bit ill equipped to go at the time since I didn’t entirely know what we needed. There are a few databases and platforms that presented that could have common threads with Arctos and TACC

This was the talk session, if it’s any indication:
Edan Bainglass: Optimizing Material Properties for Solar Energy Applications

Caleb M. Phillips: Calibration of a Hybrid Agent-based Model of Vascular Tumor Growth Using Confocal Microscopy Data

Mahdi Haddad: Poroelastic Models for Fault Reactivation in Response to Concurrent Injection and Production in Stacked Reservoirs

Meghana Palukuri: Super.Complex: A Computational Pipeline for Supervised Community Detection in Graphs

Marc E. Normandin: Towards Real-time Fully Coherent Search for Gravitational Wave Signals from Compact Binary Coalescences

William J. Beksi: 3D Scene Generation via Unsupervised Object Synthesis

Ebrahim Eslami: A Hybrid AI Hurricane Forecasting System: Deep Learning Ensemble Approach and Kalman Filter
